frontend/src/components/SchemaList.vue

```diff
@@ -1,23 +1,77 @@
 <template>
-  <ul class="list-group sticky-top">
-    <li
-      v-on:click="$emit('selectSchema', schema)"
-      v-for="schema in schemas"
-      :key="schema.id"
-      class="list-group-item"
-      :class="selectedSchema === schema ? 'active' : ''"
-    >
-      {{ schema.name }}
+  <template v-if="asDropdown">
+    <li class="nav-item dropdown">
+      <a class="nav-link dropdown-toggle" href="#" id="nav-schema-dropdown" role="button"
```

Comment:
When switching between schemas with dropdown attribute list in schema edit form doesn't change; when you switch with `Schema selection` it's okay

Reply:
Fixed. By converting the dropdown items to links the behavior is identical.

Reply:
OK, this still seems to be an issue :slightly_frowning_face:

Reply:
OK, now it's fixed.